src/util/entitlements/helpers.tsx

```diff
@@ -0,0 +1,10 @@
+export const hasEntitlement = (
+  required: string,
+  entitlements?: string[],
```

Comment:
```suggestion
  entitlement: string,
  grantedEntitlements?: string[],
```
A bit easier to read?